dev/NavigationView/NavigationView.idl

```diff
@@ -159,6 +159,10 @@ unsealed runtimeclass NavigationView : Windows.UI.Xaml.Controls.ContentControl
     [MUX_DEFAULT_VALUE("1008.0")]
     [MUX_PROPERTY_VALIDATION_CALLBACK("CoerceToGreaterThanZero")]
     Double ExpandedModeThresholdWidth { get; set; };
+    [WUXC_VERSION_PREVIEW]
```

Comment:
Should this be WUXC or MUXC?

Reply:
unfortunately it looks like our preview tag is WUXC, a hold over from OS porting days. Someday we should really rename the tags to something that makes sense.